redis/src/test/scala/zio/redis/InputSpec.scala

```diff
@@ -25,6 +26,15 @@ object InputSpec extends BaseSpec {
           } yield assert(result)(equalTo(respArgs("ABSTTL")))
         }
       ),
+      suite("Address")(
+        testM("valid value") {
+          val ip   = InetAddress.getByName("127.0.0.1")
+          val port = 42
```

Comment:
Keep these inside for comprehension as well. Simple `UIO(...)` is enough to take care of that. The same remark applies for all other tests.